pkg/log/checkpoint/checkpoint.go

```diff
@@ -47,3 +125,17 @@ func hostName() string {
 
 	return hostname
 }
+
+func logCheckPointError(logger log.Logger, err error) {
```

Comment:
(nit) I don't find this function adds clarity

Reply:
dropped the function